blackbox/docs/sql/reference/alter_table.txt

```diff
@@ -112,3 +121,56 @@ columns with a value each to identify the partition to alter.
 :value: The columns value.
 
 .. SEEALSO:: :ref:`Alter Partitioned Tables <partitioned_tables_alter>`
+
+
+Sub-commands
+============
+
+``REROUTE``
+-----------
+
+The ``REROUTE`` command provides different options to allocate a shard on a node.
+It explictly enforces and executes a reroute allocation on the cluster and makes
+it possible ot move a shard from one node to another or even allocate
+unnassigned shards to a node or cancel allocations.
+
+::
+
+    [ REROUTE reroute_option ]
+
+:shard_id: The shard id. Ranges from 0 up to the specified number of shards of
+           a table.
+
+:node_id: The node ID within the cluster.
+
+
+``REROUTE`` suports the following options to start/stop shard allocation:
+
+``MOVE``
+~~~~~~~~
+
+A started shard gets moved from one node the another. It requests a
```

Comment:
would be great if a native speaker can check these paragraphs. content-wise ok IMO

Reply:
`one node _to_ another`